### PR TITLE
Bugfix/analyses cpreq

### PR DESCRIPTION
**Pull request by PerryShafran-NOAA**
## Pull Request Testing ##

This PR does the following:

- Changes the use of cpreq to cp for files that are being copied from the /ptmp directory to the /com directory.  
- Wraps the cp of said file in a filecheck to see if the file exists before copying.  

- [ ] Describe testing already performed for this Pull Request:</br>

I ran the stats and the plots drivers and check to see if the cp of the final file was successful.  

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- Clone the tag
- Ensure that the fix directory is linked
- Run *one* of the stats jobs.  Be sure to use the qsub -v vhr as you run the stats job.  To get the final stats file to be copied to com, be sure to use qsub -v vhr=23.  You may also run for another forecast hour per your discretion, but 23 is a must to capture the cp.
- Run the plots job.  Be sure to set COMIN to emc.vpppg directory.  Check to see that the cp command worked (you can find after the tar file is created).

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

ASAP

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

File: scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh [hunk 1]
```diff
@@ -366,7 +366,9 @@ cd ${PLOTDIR}
 tar -cvf evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar *png
 
 if [ $SENDCOM = "YES" ]; then
- cpreq -v  evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar $COMOUTplots
+ if [ -s evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar ]; then
+  cp evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar $COMOUTplots
```
Comment (AliciaBentley-NOAA) [inline scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh]: @PerryShafran-NOAA Can you please make your new "cp" lines become "cp -v"? Thank you!

Comment (PerryShafran-NOAA) [reply scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh]: Done.

**Comment by ShelleyMelchior-NOAA** [comment]
I have cloned your fork and checked out branch `bugfix/analyses_cpreq`.

**Comment by ShelleyMelchior-NOAA** [comment]
I ran the rtma_ru stats jobs for vhr's 00-23. 
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_analyses_rtma_ru_grid2obs_stats.o*`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/analyses/rtma_ru.20240205`
The final stats files are comparable to the NCO EVS parallel.

**Comment by ShelleyMelchior-NOAA** [comment]
I ran the analyses plots job.
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_analyses_grid2obs_plots.o132051648`
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/analyses/atmos.20240204`
The tar file is comparable to the NCO EVS parallel.

**Comment by PerryShafran-NOAA** [comment]
@ShelleyMelchior-NOAA Everything looks good for this PR!

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
Tests yield expected results. This PR is ready for merge.

**Pull request merged**